### PR TITLE
Update vocabulary.md

### PR DESCRIPTION
**Pull request by jcstine**
A second pull request to the OVN Vocabulary, following the Vocabulary Reconciliation meeting of 2020.12.18

### CODE REVIEW

**Review by Jim1151 — COMMENTED** [review]
remove "the OVN" at the end of the entity definition
change "personage" to "Persona"
insert "verbal" into "entity's name" in the vrs definition

**Review by Jim1151 — APPROVED** [review]

**Review by Jim1151 — APPROVED** [review]

**Review by Jim1151 — APPROVED** [review]

File: vocabulary.md [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
 - **Disambiguate** - when the conversational platform hypothesizes two or more possible resolutions to a user utterance, it may ask the user for additional clarification or choose between the various interpretations to decide the user's correct intention. 
 
 
 ## E
-- **Entity** - is a part of the structured machine translation. It is a custom level data type and considered a concrete value to associate a word in a query. This is also known as annotations.
-- **Entity** -  government or private/public company that provides certain businesses or services.
+- **Entity** - the OVN uses this term in two ways.  From a technology-data perspective, is a part of the structured machine translation. It is a custom level data type and considered a concrete value to associate a word in a query. This is also known as annotations. 
+- **Entity** - the OVN uses this term in two ways.  From a business-systems perspective, it describes a person, organization, or enterprise that engages with a Dialogue System to achieve a desired outcome.
```
Comment (pmotch) [inline vocabulary.md:24]: Looks like you included two `Entity` lines here.

**Review by pmotch — CHANGES_REQUESTED** [review]
Formatting looks good. Only noticed the duplicate line for `Entity`. Once the commit is made to fix this in the PR, I'll approve and it can be merged.

File: vocabulary.md (hunk 1 above)
Comment (jcstine) [reply vocabulary.md:24]: Dan, thanks.  In our conversation right before the Holiday break, we had noted that we were using entity in two different ways -- and, after discussion and the addition of some clarifying words,  that doing so would be OK.    Please see the clarifying language -- "from a technology-data perspective" and "from a business-systems perspective"  Will this work?

**Comment by jcstine** [comment]
finishing revision as of 2021.02.18; pull request #106 no longer valid.

**Pull request closed without merging**